template.yml

```diff
@@ -164,13 +164,21 @@ Resources:
                 httpMethod: POST
                 type: aws_proxy
         components:
-          schemas:
+          AccessRight:
+            type: string
+            enum:
+              - approve_doi_request
```

Comment:
Do we have naming conventions here on enum? I usually prefer UPPER_SNAKE_CASE on enums.

Reply:
Regarding Access right, could we maybe try to align em into oauth scopes? 

And also use the convention of namespaced custom claims/scopes. 
https://auth0.com/docs/tokens/create-namespaced-custom-claims

Reply:
We probably want to have a team discussion on this, as it kinda sets the path for how easy it is to migrate to Dataporten and support Oauth etc down the road maybe?

Reply:
> Do we have naming conventions here on enum? I usually prefer UPPER_SNAKE_CASE on enums.

Actually is it case insensitive, so we can have it UPPER_CASE_SNAKE

Reply:
I will setup a meeting. But this is only a first approach on linking access rights to a role

Reply:
As a first iteration I would like to keep it simple, due to only technical reasons ( I am not sure which characters are allowed in the IAM policy tags). Once we have a working example, can change the string representation of the access right.